mriqc/interfaces/functional.py

```diff
@@ -309,6 +311,189 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class GatherTimeseriesInputSpec(BaseInterfaceInputSpec):
+    dvars = File(exists=True, mandatory=True, desc='file containing DVARS')
+    fd = File(exists=True, mandatory=True, desc='input framewise displacement')
+    mpars = File(exists=True, mandatory=True, desc='input motion parameters')
+    mpars_source = traits.Enum(
+        "FSL",
+        "AFNI",
+        "SPM",
+        "FSFAST",
+        "NIPY",
+        desc="Source of movement parameters",
+        mandatory=True,
+    )
+    outliers = File(
+        exists=True,
+        mandatory=True,
+        desc="input file containing timeseries of AFNI's outlier count")
+    quality = File(
+        exists=True,
+        mandatory=True,
+        desc="input file containing AFNI's Quality Index")
+
+
+class GatherTimeseriesOutputSpec(TraitedSpec):
+    timeseries_file = File(
+        exists=True,
+        desc='output confounds file',
+        resolve=True,
+        extensions="tsv")
+    timeseries_metadata = traits.Dict
```

Comment:
`exists=True` on outputs just leads to nasty exceptions when something fails. And we should describe the dictionary.

```suggestion
    timeseries_file = File(desc='output confounds file')
    timeseries_metadata = traits.Dict(desc='Metadata dictionary describing columns')
```
```